ios/FluentUI.xcodeproj/project.pbxproj

```diff
@@ -1659,6 +1659,7 @@
 			buildSettings = {
 				CLANG_ENABLE_MODULES = YES;
 				CODE_SIGN_IDENTITY = "";
+				"CODE_SIGN_IDENTITY[sdk=macosx*]" = "-";
```

Comment:
You don't need to change the project file, do you?

Reply:
oh no. not sure how it crept in...sorry i will revert this